Sources/Extensions/Array/Array+Extensions.swift

```diff
@@ -70,7 +70,7 @@ public extension Array {
 
     // Subscript for safe access to element by index
     subscript(safe index: Index) -> Element? {
-        return index < count ? self[index] : nil
+        return indices.contains(index) ? self[index] : nil
```

Comment:
Think it would be better to additionally check index for being non-negative.
Subscripts should be O(1) in random access collections

Reply:
Actually, the main reason of this change is to check for negative indices..

Reply:
I got it ) 
I mean better add another condition, so that subscript stays O(1)

Reply:
I got it )
I mean better add another condition, so that subscript stays O(1)